string-constants-lib/string-constants/private/danish-string-constants.rkt

```diff
@@ -1243,6 +1291,175 @@ please adhere to these guidelines:
   (gui-tool-show-gui-toolbar "Vis GUI-værktøjslinjen")
   (gui-tool-hide-gui-toolbar "Skjul GUI-værktøjslinjen")
   (gui-tool-insert-gui "Indsæt GUI")
+
+  ;; contract violation tracking
   
+  ; tooltip for new planet icon in drscheme window (must have a planet violation logged to see it)
+  (show-planet-contract-violations "Vis PLaneT kontraktovertrædelser")
+
+    
+  ;; planet status messages in the bottom of the drscheme window; the ~a is filled with the name of the package
+  (planet-downloading "PLaneT: Downloader ~a…")
+  (planet-installing "PLaneT: Installerer ~a…")
+  (planet-finished "PLaneT: Færdig med ~a.")
+  (planet-docs-building "PLaneT: Bygger dokumentation (udløst af ~a)…")
+  (planet-no-status "PLaneT") ;; this can happen when there is status shown in a different and then the user switches to a tab where planet hasn't been used
+  
+  (bug-report-field-pkg "Information om pakkehåndteringssystemet")
+  
+  ;; string normalization. To see this, paste some text with a ligature into DrRacket
+  ;; the first three strings are in the dialog that appears. The last one is in the preferences dialog
+  (normalize "Normalisér")
+  (leave-alone "Lad være")
+  (normalize-string-info "Strengen, som du indsat, indeholder ligaturer eller andre ikke-normliserede tegn. Normalisér dem?")
+  (normalize-string-preference "Normalisér indsatte strenge")
+  (ask-about-normalizing-strings "Spørg før strengnormalisering")
+  
+  (always-use-platform-specific-linefeed-convention "Brug altid den platformspecifikke linjeendelseskonvention")
+
+  ;; optimization coach
+  (hide-optimization-coach "Skjul optimeringscoachen")
+  (show-optimization-coach "Vis optimeringscoachen")
+
+  ;; labels used (in a big font) in the background of the definitions and interactions windows
+  (definitions-window-label "definitioner")
+  (interactions-window-label "interaktioner")
+  (hide-defs/ints-label "Skjul definitioner/interaktioner mærkater") ;; popup menu
+  (show-defs/ints-label "Vis definitioner/interaktioner mærkater") ;; preferences checkbox
+
+  ;; menu item in the 'edit' menu; applies to editors with programs in them
+  ;; (technically, editors that implement color:text<%>)
+  (spell-check-string-constants "Stavekontrol i konstante strenge")
```

Comment:
I think "Stavekontrol i strengkonstanter" is better.

Reply:
Yeah, that is better. Feels less operational and more syntactic.